[2022-12-12 12:48:39,673] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.transfer_data manual__2022-12-12T12:46:40.745106+00:00 [queued]>
[2022-12-12 12:48:40,599] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: prepare_data.transfer_data manual__2022-12-12T12:46:40.745106+00:00 [queued]>
[2022-12-12 12:48:40,599] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-12 12:48:40,599] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-12 12:48:40,599] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-12 12:48:41,730] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): transfer_data> on 2022-12-12 12:46:40.745106+00:00
[2022-12-12 12:48:41,782] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'prepare_data', 'transfer_data', 'manual__2022-12-12T12:46:40.745106+00:00', '--job-id', '377', '--raw', '--subdir', 'DAGS_FOLDER/prepare_data.py', '--cfg-path', '/tmp/tmpy6a1dj4s', '--error-file', '/tmp/tmpy5fudjql']
[2022-12-12 12:48:41,809] {standard_task_runner.py:80} INFO - Job 377: Subtask transfer_data
[2022-12-12 12:48:41,733] {standard_task_runner.py:52} INFO - Started process 1171 to run task
[2022-12-12 12:48:44,824] {task_command.py:369} INFO - Running <TaskInstance: prepare_data.transfer_data manual__2022-12-12T12:46:40.745106+00:00 [running]> on host b778137d8ea6
[2022-12-12 12:48:49,672] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=prepare_data
AIRFLOW_CTX_TASK_ID=transfer_data
AIRFLOW_CTX_EXECUTION_DATE=2022-12-12T12:46:40.745106+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-12T12:46:40.745106+00:00
[2022-12-12 12:48:50,197] {base.py:68} INFO - Using connection ID 'postgres_default' for task execution.
[2022-12-12 12:48:51,228] {python.py:173} INFO - Done. Returned value was: None
[2022-12-12 12:48:51,604] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=prepare_data, task_id=transfer_data, execution_date=20221212T124640, start_date=20221212T124839, end_date=20221212T124851
[2022-12-12 12:48:52,195] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-12 12:48:52,975] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
